src/Config.php

```diff
@@ -112,6 +112,13 @@ public static function preItemUpdate(CommonDBTM $item): CommonDBTM
             }
         }
 
+        // Handle use_parent_entity field
+        if (!isset($item->input['use_parent_entity'])) {
+            $item->input['use_parent_entity'] = 0;
+        } elseif ($item->input['use_parent_entity'] == 'on') {
+            $item->input['use_parent_entity'] = 1;
+        }
```

Comment:
`use_parent_entity` should be a boolean?! _`true`/`false` instead of `1`/`0`_
```suggestion
        $item->input['use_parent_entity'] = ($item->input['use_parent_entity'] ?? '') === 'on';
```